src/net/gosha/atproto/jetstream.clj

```diff
@@ -0,0 +1,114 @@
+(ns net.gosha.atproto.jetstream
+  (:require
+   [charred.api           :as json]
+   [clojure.core.async    :as async]
+   [clojure.string :as string]
+   [clojure.tools.logging :as log])
+  (:import
+   [java.net URI]
+   [org.java_websocket.client WebSocketClient]))
```

Comment:
I was having a `ClassNotFoundException` exception in my REPL unless I added this to `deps.edn`:

```clojure
org.java-websocket/Java-WebSocket {:mvn/version "1.6.0"}
```

Do we need to depend on it explicitly, or is this some JDK config issue on my side?